streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -766,6 +769,24 @@ void runOnce() {
         return records;
     }
 
+    private OffsetResetStrategy getResetStrategy(final TopicPartition partition) {
```

Comment:
Wondering if we should reuse this method within `StreamThread#resetInvalidOffsets`?

Reply:
Yeah, I think we should.